src/components/date-picker/DatePicker.vue

```diff
@@ -127,9 +127,25 @@ export default class DatePicker extends DateMixin {
     this.date = this.yyyyMmDdToDate(this.initialValue)
   }
 
+  /** The formatted picker date string (YYYY-MM-DD) */
+  get pickerDate (): string {
```

Comment:
Please add a comment indicating you need to do this because the picker returns a Date object with 00:00 UTC.

Reply:
Perfect 👍